lang/cs/Settings.StyleCop

```diff
@@ -682,146 +682,146 @@ under the License.
       <Rules>
         <Rule Name="KeywordsMustBeSpacedCorrectly">
           <RuleSettings>
-            <BooleanProperty Name="Enabled">False</BooleanProperty>
+            <BooleanProperty Name="Enabled">True</BooleanProperty>
```

Comment:
Default is `False`. For the `False` rules, we just remove that. 
But, I think @anupam128 made this file to show what rules exists.